app/src/main/java/com/vultisig/wallet/ui/models/keysign/KeysignFlowViewModel.kt

```diff
@@ -208,12 +210,13 @@ internal class KeysignFlowViewModel @Inject constructor(
     ) {
         try {
             _currentVault = vault
-            _keysignPayload = keysignPayload
+            val modifiedKeysignPayload = updateSolanaKeysignPayload(keysignPayload)
```

Comment:
I think this will cause a keysign issue , since the latest blockhash is used to generate the transaction